src/megaclient.cpp

```diff
@@ -1211,6 +1211,35 @@ MegaClient::~MegaClient()
     delete dbaccess;
 }
 
+std::string &MegaClient::getPublicLink(bool newLinkFormat, int type, handle ph, const char *key)
+{
+    string strlink = "https://mega.nz/";
+    string nodeType;
+    if (newLinkFormat)
+    {
+        nodeType = (type ? "file/" : "folder/");
+    }
+    else
+    {
+        nodeType = (type ? "#!" : "#F!");
+    }
+
+    strlink += nodeType;
+
+    char *base64ph = new char[12];
+    Base64::btoa((byte*)&(ph), MegaClient::NODEHANDLE, base64ph);
+    strlink += base64ph;
```

Comment:
Not a biggie, but here we should start using the new
```Base64Str<NODEHANDLE>((const byte*)&ph)```

Reply:
Fixed it